go_backend/main.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"context"
+
+	"github.com/gin-contrib/cors"
+	"github.com/gin-gonic/gin"
+	"github.com/go-redis/redis/v8"
+)
+
+var ctx = context.Background()
+
+var Rdb = redis.NewClient(&redis.Options{
+	Addr: "redis:6379",
+	DB:   0,
+})
+
+func main() {
+	app := gin.Default()
+	app.Use(cors.Default())
```

Comment:
What does this do? Is it open for everyone?

Reply:
Yes. All origins are allowed. I should probably set up the dev and prod settings for it tbh.

Reply:
Origins are now set based on the app environment